synapse/python_dependencies.py

```diff
@@ -66,8 +66,8 @@
     "msgpack>=0.5.2",
     "phonenumbers>=8.2.0",
     "prometheus_client>=0.0.18,<0.9.0",
-    # we use attr.validators.deep_iterable, which arrived in 19.1.0
-    "attrs>=19.1.0",
+    # we use attrs `order` param, which arrived in 19.2.0
+    "attrs>=19.2.0",
```

Comment:
I think this should be fine, but I can also implement the comparison functions manually (its just so much more verbose).

Reply:
Looks like we checked with packagers and no one had an issue with it.